### PR TITLE
Readme rewrite

### PR DESCRIPTION
**Pull request by lunacodes**
Draft Pull Request for Readme rewrite.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,119 @@
...
-* Move progress bar above controls?
-* Change background color on Play, to avoid whitewash
+* GitHub Repo: https://github.com/common-workflow-language/
+* Staging Branch: https://github.com/common-workflow-language/cwl-website/tree/add-jekyll-site
+* Staging Site: https://www.commonwl.org/cwl-staging/
 
-### Subtitles
+<a id="vision-for-the-cwl-project"></a>
+## Vision for the CWL Project
+_What this community hopes to have accomplished in the future_
```
Comment (mr-c) [inline README.md:25]: Yay! Though the mission and vision statements shouldn't be part of this PR, and they need to be part of the content of www.commonwl.org and not the README for this repo

File: README.md [hunk 2]
```diff
@@ -1,27 +1,119 @@
...
+<a id="editing-the-site---general-flow"></a>
+
+<a id="editing-the-site---general-flow"></a>
+## Editing the Site - General Flow
+
+1. On `main`, run a `git fetch --all` and pull in any changes.
+2. Switch to `add-jekyll-site` and then merge the changes via `git merge main`
+3. Make sure you're making your changes on the `add-jekyll-site` (or your own branch, which you can merge to `add-jekyll-site`).
+4. Push to `add-jekyll-site` and then check that everything looks good. 
+5. If all is well, head over to the [main repo](https://github.com/common-workflow-language/) and create a Pull Request.
```
Comment (mr-c) [inline README.md:52]:
```suggestion
5. If all is well, head over to the [main repo](https://github.com/common-workflow-language/cwl-website) and create a Pull Request.
```

File: README.md (hunk 1 above)
Comment (lunacodes) [reply README.md:25]: Ahhhh whoops - that makes more sense! I created a PR (https://github.com/common-workflow-language/cwl-website/pull/62) for it, and will delete from here.

Comment (mr-c) [reply README.md:25]: That PR was merged, so please delete the mission and vision statement from here.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,119 @@
...
+
+```yaml
+title: CWL Local Site
+url: "cwl.test"
+baseurl: ""
+port: 80
+```
+
+The command for this is: `bundle exec jekyll serve --config "_config.yml,_config_local.yml"`. (Note the lack of spaces between the two config files).
+
```
Comment (mr-c) [inline README.md:69]:
```
$ bundle exec jekyll serve --config "_config.yml"
Could not locate Gemfile or .bundle/ directory
```

Comment (mr-c) [reply README.md:69]: For Debian, I had install the following packages: ruby-dev, ruby-bundler ; that should probably be mentioned somewhere

Comment (mr-c) [reply README.md:69]: Okay, I figured out that I have to run `bundle` commands from the `content` directory..

File: README.md [hunk 4]
```diff
@@ -1,27 +1,119 @@
...
+
+<a id="directory-structure"></a>
+## Directory Structure
+
+As of March 11 2021, the directory structure looks as follows
+
+```sh
+├── _data
+├── _includes
```
Comment (mr-c) [inline README.md:80]: None of these directories are in the root of this repository, they are in the `content` directory...

Comment (lunacodes) [reply README.md]: Updated this. Thx

**Review by mr-c — APPROVED** [review]
